custom/icds_reports/views.py

```diff
@@ -121,7 +121,7 @@
 
 
 @location_safe
-@method_decorator([toggles.ICDS_REPORTS.required_decorator(), login_and_domain_required], name='dispatch')
+@method_decorator([login_and_domain_required], name='dispatch')
```

Comment:
It's likely no one uses this, but will go through nginx logs before I remove the redirect